src/magicbox/InputManager.ts

```diff
@@ -261,6 +267,18 @@ export class InputManager {
     }
   }
 
+  private handleLeftRightArrow(e: KeyboardEvent) {
+    if ($$(this.root).find(`.${Component.computeCssClassNameForType('QuerySuggestPreview')}`)) {
```

Comment:
Is there a way to find the component without the root or the searchInterface because now, I need to pass the root to this class to find the component.

Reply:
I think the root is the safest. Using `window` would not be as good because there might be multiple search interfaces in a single page.